src/EventStore.ClusterNode/ClusterNodeOptions.cs

```diff
@@ -152,6 +152,8 @@ public class ClusterNodeOptions : IOptions
         public int WorkerThreads { get; set; }
         [ArgDescription(Opts.ProjectionsQueryExpiryDescr, Opts.ProjectionsGroup)]
         public int ProjectionsQueryExpiry { get; set; }
+        [ArgDescription(Opts.FailOutoforderProjectionsDescr, Opts.ProjectionsGroup)]
+        public bool FailOutoforderProjections { get; set; }
```

Comment:
Nit: to keep the terminology the same as projections, I think this should be FaultOutOfOrderProjections, also the casing is inconsistent

Reply:
done